### DISCUSSION TITLE
v0.3.9

### DISCUSSION THREAD

**Discussion by stefansundin** [stefansundin/privatkopiera#176] [Announcements]
- Fixade TV4 Play. Tack Viktor (#175).
- Du kan nu ställa in vilken filändelse som används som standard.

**Full Changelog**: https://github.com/stefansundin/privatkopiera/compare/v0.3.8...v0.3.9

<hr /><em>This discussion was created from the release <a href='https://github.com/stefansundin/privatkopiera/releases/tag/v0.3.9'>v0.3.9</a>.</em>

**Comment by 83sixten** [comment]
Hej Stefan, 8-) Är mycket glad och tacksam för att du startade härliga projektet privatkopiera, för jag gillar  programet extremt mycket. Jag körde först med FFmpeg_20170503_Win32_Static men använder nu den nyaste versionen av FFmpeg

/ / ffmpeg version N-110002-g7c130d6911-20230313 Copyright (c) 2000-2023 the FFmpeg developers
built with gcc 12.2.0 (crosstool-NG 1.25.0.90_cf9beb1) / /

och det fungerar väldigt bra även med den nya versionen. Eftersom du startade med gå ifrån suffix .mp4 till .mkv så har jag även testat med att få in text i .mkv filen. Vilket fungerar klockrent till att få in texten vid det sevärda programmet man vill se med text, eftersom i dagsläget blir det text på rapport som förmodligen har kvar den gamla kodningen av svtplay sidan. 

Eftersom du inte har haft eller fått någon ny individ som har velat betala din programerings tid, till att återställa möjligheten att få text till övriga programmen, berättar jag nu för dig att texten fungerar fint till programmet rapport, vilket jag använder bl. a. till rapport som sänds vid 19:30 tiden.

Och jag gissar på att det kommer att bli text vid införandet av detta kommando (som är kopplat till ffmpeg version N-110002-g7c130d6911-20230313 Copyright (c) 2000-2023 the FFmpeg developers built with gcc 12.2.0 (crosstool-NG 1.25.0.90_cf9beb1) || alltså kommandot -stag fourcc/tag || om du återinför kodningen som kan ge individen möjligheten att välja text till programmet, och kommer då att bli text i .mkv filen istället för en srt fil till suffixet .mp4.

-acodec copy -vcodec copy -stag fourcc/tag

Man kan även använda kommandot || -scodec copy || och används när man vill göra om filen, || ifrån 720p till 360p vilket innebär att man kopierar textet som finns i filen till den nya filen. ||

Ha det gott och bra, samt så tackar jag dig otroligt mycket för ett sådant härligt, och ett bra program, privatkopiera. 

Om du hade velat haft någonting av mig, (ett mail konto på // https://login.live.com/login.srf? // med ett godkänt konto som är gjort med falskt personnummer (( vilket kommer att vara giltigt )) kan du kanske se det som ett stort tack ifrån mig med tanke på dina härliga kunskaper inom din programmering.

Skicka ett mail på sixten83@outlook.com om du kanske vill ha någon som hade hjälp dig att testa ditt härliga program, men jag kontrollerar den inte varje dag, det kan ta max tre dagar innan en kontroll, för på den här adressen finns det extremt mycket mer att inhämta med ett konto för att få se på tv-program. https://arenan.yle.fi/tv

Ha det extemt gott och bra Stefan
Mvh Chrille

**Reply by stefansundin** [reply]
Hej Chrille,

Jag tror att du stöter på problemet som är beskrivet i #120. Kan du testa med FFmpeg från https://github.com/yt-dlp/FFmpeg-Builds#downloads och see om det löser ditt problem?

**Comment by 83sixten** [comment]
Tjena Stefan :^-)

Jag har precis provat på något som jag anser är strunt, och gjorde precis ett test på
"Flykten till Östermalm - Avsnitt 1.mkv"

[https://www.svtstatic.se/image/custom/1024/39147231/1686130593?format=auto&chromaSubSampling=false&enableAvif=true]<https://www.svtplay.se/video/jEWMwMP/flykten-till-ostermalm/avsnitt-1?id=jEWMwMP>
Flykten till Östermalm – Avsnitt 1<https://www.svtplay.se/video/jEWMwMP/flykten-till-ostermalm/avsnitt-1?id=jEWMwMP>
Familjen Mursa försöker finna sig i de problem deras dyra paradvåning på Östermalm innebär och dottern Melissa försöker lära sig rikssvenska för att passa in i Stockholm. I Sven Melanders sista komediserie följer vi familjen Mursa som efter att de ärvt hundratals miljoner kronor flyttar från Rosengård i Malmö till Strandvägen i Stockholm. Det är en helt ny värld... I rollerna: Özz Nûjen, Filip Dikmen, Anders Jansson, Reuben Sallmander, Selma Caglar, Yussra El Abdouni m.fl. Del 1 av 8.
www.svtplay.se

Och provade med att köra med kommandot / -acodec copy -vcodec copy -stag fourcc/tag /
och jag fick text, "trots att det står i privatkopiera under fliken ström", "text closed.vtt".
Och ändå får jag text. Mycket imponerande, måste jag säga. Men det står inte detta när jag
kör på rapport. Oavsett så tackar jag dig ofantligt mycket för ett sådant här underbart
program.

Backar vi tillbaka i tiden så satt jag och hade alla tv-kanaler i Sverige, Norge, Finland, Frankrike,
Danmark, och många fler länder inom Eurocrypt, S / M / S2 och S3 inom tripple des.
Det var tider det 8^-) Idag ser jag faktiskt inte mer än det som svtplay erbjuder, det räcker
tycker jag. 😉

Jag tackar dig extremt mycket för din hjälp inom den här saken, och tycker att det är
härligt att du gav mig din tid och en fantastisk och underbar hjälp. Tackar mycket för det.

Skicka mig en notering om du känner för det inom det som jag skrev. "Jag själv är inte ute
efter att skada någon eller någoting inom det jag kan", jag lär mig sakerna bara för att själv
skall förstå mer inom olika kunskaper, så att jag själv kan lära mig och förstå:
 "att bygga upp den totala säkerheten inom olika vinklar där andra har en möjlighet
att frambringa en osäkerhet om man själv inte bejakar de situationer man befinner sig i".

Tack än engång Stefan
Mvh Christer
________________________________
Från: Stefan Sundin ***@***.***>
Skickat: den 5 juli 2023 20:31
Till: stefansundin/privatkopiera ***@***.***>
Kopia: 83sixten ***@***.***>; Comment ***@***.***>
Ämne: Re: [stefansundin/privatkopiera] v0.3.9 (Discussion #176)


Hej Chrille,

Jag tror att du stöter på problemet som är beskrivet i #120<https://github.com/stefansundin/privatkopiera/issues/120>. Kan du testa med FFmpeg från https://github.com/yt-dlp/FFmpeg-Builds#downloads och see om det löser ditt problem?
